app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def auth_user
```

Comment:
Let's use `authenticate_user` instead of `auth_user` because auth means authorize and authenticate at the same time. It might be misleading.

Reply:
done